src/draw/lv_image_dsc.h

```diff
@@ -38,43 +34,38 @@ typedef enum _lv_image_flags_t {
      * For RGB map of the image data, mark if it's pre-multiplied with alpha.
      * For indexed image, this bit indicated palette data is pre-multiplied with alpha.
      */
-    LV_IMAGE_FLAGS_PREMULTIPLIED    = (1 << 0),
-
-    /**
-     * If the image data is malloced and can be processed in place.
-     * In image decoder post processing, this flag means we modify it in-place.
-     */
-    LV_IMAGE_FLAGS_MODIFIABLE       = (1 << 1),
-
-    /**
-     * Indicating it's a vector image instead of default raster image.
-     * Some of the flags are not usable for vector image, like PREMULTIPLIED.
-     */
-    LV_IMAGE_FLAGS_VECTORS          = (1 << 2),
-
+    LV_IMAGE_FLAGS_PREMULTIPLIED    = 0x01,
     /**
      * The image data is compressed, so decoder needs to decode image firstly.
      * If this flag is set, the whole image will be decompressed upon decode, and
      * `get_area_cb` won't be necessary.
      */
-    LV_IMAGE_FLAGS_COMPRESSED       = (1 << 3),
+    LV_IMAGE_FLAGS_COMPRESSED       = 0x08,
+
+    /*Below flags are applicable only for draw buffer header.*/
 
     /**
      * The image is allocated from heap, thus should be freed after use.
      */
-    LV_IMAGE_FLAGS_ALLOCATED        = (1 << 4),
+    LV_IMAGE_FLAGS_ALLOCATED        = 0x0100,
```

Comment:
There is one more `0` in value.
It should be: `0x0010`

Reply:
This is intended to leave some bit available for `const image` flags.
The `ALLOCATED` bit and below ones are applicable only for draw buffer header.

Reply:
But then `LV_IMAGE_FLAGS_ALLOCATED` and `LV_IMAGE_FLAGS_USER1` will have same value.
Same with `LV_IMAGE_FLAGS_MODIFIABLE` and `LV_IMAGE_FLAGS_USER2`

Reply:
Ah, I see. Will fix it now.